packages/env/lib/create-docker-compose-config.js

```diff
@@ -1,11 +1,17 @@
+// Gets the dependency volume mapping (including the current context)
+function getDependencyConfig( deps, currentContext ) {
+	const dependencies = [ ...deps, currentContext ];
+	return dependencies.map( ( { path, pathName, type } ) => `      - ${ path }/:/var/www/html/wp-content/${ type }s/${ pathName }/\n` ).join( '' );
+}
+
```

Comment:
This isn't really useful anywhere else, specially with the dependency on whitespace amount. I think we should just inline it.